pkgs/top-level/all-packages.nix

```diff
@@ -31666,12 +31666,13 @@ with pkgs;
       };
     };
   };
-  gnuradio3_8 = disable-warnings-if-gcc13 (callPackage ../applications/radio/gnuradio/wrapper.nix {
```

Comment:
In my local testings I noticed that if `disable-warnings-if-gcc13` would have been applied to the unwrapped derivation, it would have worked. @chuangzhu do you have strong preference to using gcc12 vs disabling gcc13 warnings?

Reply:
Interesting, it failed in my local test https://gist.github.com/chuangzhu/658ab70024fea247959c5f00aa914fee

Reply:
> Interesting, it failed in my local test [gist.github.com/chuangzhu/658ab70024fea247959c5f00aa914fee](https://gist.github.com/chuangzhu/658ab70024fea247959c5f00aa914fee)

I was mistaken, sorry about that (confused GR3.9 and GR3.8)...

Even if I'll revert this to your version of the PR, I wonder how can we fix the Darwin build errors.. According to [this successful hydra build](
https://nydra.nixos.org/build/247654168), and it's nixpkgs revision, the Darwin build didn't fail there thanks to `disable-warnings-if-gcc13` (I also verified the default `gcc --version` is 13.2.0 there. I wonder, why did you use `stdenvAdapters.useLibsFrom`, and not simply `gcc12Stdenv`?

Reply:
I guess Darwin uses LLVM so it's not affected by the GCC update. For Linux however, there seems to be some ABI incompatibility between boost and GCC 12's libstdc++